server/src/user-org/user-org.controller.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+  HttpException,
+  HttpStatus,
+} from '@nestjs/common';
+import { DeleteResult } from 'typeorm';
+
+import { UserOrganization } from './entities/user-org.entitiy';
+import { UserOrganizationsService } from './user-org.service';
+import { CreateUserOrganizationDto } from './dto/create-user-org.dto';
+import { UpdateUserOrganizationDto } from './dto/update-user-org.dto';
+
+@Controller('userOrganizations')
+export class UserOrganizationsController {
+  constructor(private readonly userOrganizationsService: UserOrganizationsService) {}
+
+  @Post()
+  async create(
+    @Body() createUserOrganizationsDto: CreateUserOrganizationDto,
+  ): Promise<UserOrganization> {
+    try {
+      const userOrg = await this.userOrganizationsService.create(createUserOrganizationsDto);
+      return userOrg;
+    } catch (error) {
+      throw new HttpException(
+        { status: HttpStatus.CONFLICT, message: `${error}` },
+        HttpStatus.CONFLICT,
+      );
+    }
+  }
+
+  @Get()
+  async findAll(): Promise<UserOrganization[]> {
+    const allUserOrgs = await this.userOrganizationsService.findAll();
+    return allUserOrgs;
+  }
+
+  @Get(':id')
+  async findOne(@Param('id') id: string): Promise<UserOrganization> {
```

Comment:
#### Fix 
Ignore if id is in fact a string

For all the controller methods using an id that should be an int use the `ParseIntPipe`, and follow the example in [NestJs](https://docs.nestjs.com/pipes)